presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
```

Comment:
Also, in the highlights, we don't need to copy paste from below and can rephrase. For example here - we can just say something like 

```
Add functions func1, func2, func3
```